CSS/styles.css

```diff
@@ -0,0 +1,47 @@
+body {
+ 
+    color: rgb(221, 221, 221);
+    background-color: rgb(37, 37, 117);
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+#whole{
+    margin: 50px;
+}
+
+#form{
+    background-color: rgb(36, 35, 35);
+    margin: 20%; 
+    margin-top: 5%;
+    color: white;
+    width: 70%;
+    /*text-align: center; */
```

Comment:
Please do not leave unused code as a comment on your file, as it will make your code less clean.